db/sql/migrations/updates/20180626001337_alter_mem_round.sql

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+
+/*
+  DESCRIPTION: Alter 'mem_round' table - change data type of 'round' from BIGINT to INT.
+
+  PARAMETERS: None
+*/
+
+ALTER TABLE mem_round ALTER COLUMN round TYPE INT;
```

Comment:
Please update the migration file name to more meaning full name e.g. `alter_round_in_mem_rounds` or `change_round_type_in_mem_rounds`.